RadioTransmission_ST/RadioChannel.py

```diff
@@ -15,6 +15,7 @@ def __calcSignalPow(self, signal):
         return sigPow
 
     def transmit(self, inputSignal, snr=None, signalOffset=0, channelAttenuation=1, freqErr=0, phaseErr=0, adcSamplingErr=None):
+
```

Comment:
????

Reply:
xD to pokozstałości po konwersji up/down :D te czyste xd

Reply:
to usuń to xd po co śmieci zostawiać w repo